packages/xo-web/src/common/xo/index.js

```diff
@@ -376,7 +376,7 @@ export const dismissNotification = id => {
 export const subscribeNotifications = createSubscription(async () => {
   const { user, xoaUpdaterState } = store.getState()
   if (
-    process.env.XOA_PLAN === 5 ||
+    +process.env.XOA_PLAN === 5 ||
```

Comment:
Could we use `utils/getXoaPlan` to centralize the PLANS values?

Reply:
Also, I think we should declare constants for easier comparisons: `+process.env.XOA_PLAN === XOA_PLAN_SOURCES`.

Reply:
I agree with you :+1: maybe in its dedicated module `plans` which will contains all plans value ?

Reply:
Yes, and maybe it should export the plan as a number as well.

Reply:
The issue with `utils/getXoaPlan` is that it returns a string, which is not great for lower/greater comparisons. I agree there may be a better way to handle plan checks, though, but it wasn't this PR's goal.